evalai/download_starters.py

```diff
@@ -0,0 +1,65 @@
+import click
+import os
+import requests
+import tempfile
+import zipfile
+
+from click import echo, style
+
+from evalai.utils.config import STARTERS_ZIP_URL
+
+
+@click.group(invoke_without_command=True)
+def download_evalai_starters_kit():
```

Comment:
@takitsuse I think you need to update here also.

Reply:
I will make those changes, but maybe we should hear from @RishabhJain2018 because his review conflicts with this.

Reply:
And could you have a look at PR# 2512? I have commented on your review.